scripts/convert.js

```diff
@@ -16,6 +16,8 @@ const indicators = Object.entries(INDICATORS).map(([key, indicator]) => ({
 getStdin().then((rawCsv) => {
   csv.parse(rawCsv, { columns: true }, (err, data) => {
     const cities = data.map((row) => {
+      if (!row.Cities) return null;
```

Comment:
I guess this (accompanied by the `filter` call below) excludes any rows without a city name, right? Is that a scenario we've encountered?

Reply:
Yeah I encountered it this time around. I don't know if that's because this time I went via Excel instead of straight out of Google sheets.